src/test/java/com/snowflake/kafka/connector/SnowflakeSinkTaskForStreamingIT.java

```diff
@@ -162,31 +162,39 @@ public void testTaskToClientMapping() throws Exception {
 
   @Test
   public void testMultipleSinkTaskWithLogs() throws Exception {
-    int partition0 = 0;
-    int partition1 = 1;
-
     // setup log mocking for task1
     MockitoAnnotations.initMocks(this);
     Mockito.when(logger.isInfoEnabled()).thenReturn(true);
     Mockito.when(logger.isDebugEnabled()).thenReturn(true);
     Mockito.when(logger.isWarnEnabled()).thenReturn(true);
 
+    // TODO @rcheng: factor these out, has been a pain updating this test with the dupe code...
```

Comment:
Planning on pushing a new PR (very soon like today or monday) for this TODO. Decided to split PRs since the issues are separate. Happy to include it here if anyone has strong opinions otherwise